src/js/models/v2/agencyV2/BaseAgencyOverview.js

```diff
@@ -10,6 +10,11 @@ const BaseAgencyOverview = {
         this.website = data.website || '';
         // eslint-disable-next-line camelcase
         this.id = data.agency_id || '';
+        this.logo = data.icon_filename || '';
+        this.mission = data.mission || '--';
+        this.congressionalJustification = data.congressional_justification_url || '';
+        this.showAboutData = data.about_agency_data || false;
+        this.subtierCount = data.subtier_agency_count || 0;
```

Comment:
Should falsey here be converted to 0? Might that be misleading? Or will it never happen?

Reply:
`subtier_agency_count` is listed as required in the [API contract](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/agency/toptier_code.md), so it shouldn't happen